docs/build.sh

```diff
@@ -160,6 +162,10 @@ rsync -av \
       "${SCRIPT_DIR}"/_ext \
       "${GENERATED_RST_DIR}"
 
+# Merge generated redirects
+cat /tmp/redirects.txt >> "${GENERATED_RST_DIR}"/redirects.txt
```

Comment:
think of a better way than /tmp/redirects

Reply:
ive added a todo to move the generation of the redirects file to bazel - for now this will do i think